src/codecs/png.rs

```diff
@@ -193,6 +193,27 @@ impl<R: Read> PngDecoder<R> {
         Ok(PngDecoder { color_type, reader })
     }
 
+    /// Returns the gamma value of the image or None if no gamma value is indicated.
+    ///
+    /// If an sRGB chunk is present this method returns a gamma value of 0.45455 and ignores the
+    /// value in the gAMA chunk. This is the recommended behavior according to the PNG standard:
+    ///
+    /// > When the sRGB chunk is present, [...] decoders that recognize the sRGB chunk but are not
+    /// > capable of colour management are recommended to ignore the gAMA and cHRM chunks, and use
+    /// > the values given above as if they had appeared in gAMA and cHRM chunks.
+    pub fn gamma_value(&self) -> ImageResult<Option<f64>> {
+        let gamma = if self.reader.info().srgb.is_some() {
+            Some(0.45455)
```

Comment:
I wonder why you have chosen to manually consider `srgb` and `gama_chunk`, instead of using [Info::source_gamma](https://docs.rs/png/0.17.10/png/struct.Info.html#structfield.source_gamma), which AFAICT takes both `srgb` ([here](https://github.com/image-rs/image-png/blob/cbea7ffd093573a76a406a145d776acf7a1d5baa/src/decoder/stream.rs#L1078)) and `gama_chunk` ([here](https://github.com/image-rs/image-png/blob/cbea7ffd093573a76a406a145d776acf7a1d5baa/src/decoder/stream.rs#L1052)) into account.  FWIW, [`png::srgb::substitute_gamma`](https://github.com/image-rs/image-png/blob/cbea7ffd093573a76a406a145d776acf7a1d5baa/src/srgb.rs#L6) is also `45455`

Reply:
That's a good point point... I didn't notice the `png` crate already implemented that logic